inst/pages/21_microbiome_community.qmd

```diff
@@ -43,17 +43,17 @@ plotAbundance(tse_phylum, assay.type="relabundance", rank = "Phylum",
 
 ### Composition heatmap {#sec-composition-heatmap}
 
-Community composition can be visualized with heatmap, where the
-horizontal axis represents samples and the vertical axis the
-taxa. The color of each intersection point represents abundance of a taxon
+Community composition can be visualized with a heatmap, where the
+horizontal axis represents samples and the vertical axis represents
+taxa. The color of each intersection point represents the abundance of a taxon
 in a specific sample.
 
 Here,  abundances are  first CLR  (centered log-ratio)  transformed to
-remove  compositionality bias. Then  Z  transformation  is applied  to
+remove  compositionality bias. Then, Standardize(or  'Z') transformation  is applied  to
```

Comment:
Standardize(or  'Z') --> "Then, standardization is applied..."

Reply:
Done :)

Reply:
resolve conversations when they are done. This way we can keep track on what has been dealt with.